src/DropdownTrigger/index.scss

```diff
@@ -45,9 +46,6 @@
 .nds-dropdownTrigger-value {
   color: var(--font-color-primary);
   max-width: 100%;
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
```

Comment:
1. This wasn't actually working as expected
2. It's best not to truncate the value we render when an item is selected, e.g. two accounts with the same start words in their name